infra/build/Makefile

```diff
@@ -28,7 +28,9 @@ TERRAGRUNT_VERSION := 0.23.25
 KUSTOMIZE_VERSION := 3.6.0
 
 DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.13.1
-DOCKER_TAG_MAJOR_VERSION_DEVELOPER_TOOLS = $(firstword $(subst ., , $(DOCKER_TAG_VERSION_DEVELOPER_TOOLS)))
+DOCKER_TAG_MAJOR_VERSION_DEVELOPER_TOOLS := $(firstword $(subst ., , $(DOCKER_TAG_VERSION_DEVELOPER_TOOLS)))
+DOCKER_TAG_MINOR_VERSION_DEVELOPER_TOOLS := $(shell echo "${DOCKER_TAG_VERSION_DEVELOPER_TOOLS}" | awk -F. '{print $$1"."$$2}')
```

Comment:
if 0.12.1 -> 0.12 is expected shouldn't this be ` awk -F. '{print $1"."$2}'`?

Reply:
Makefiles need escaping.